contracts/contracts/rem.system/src/rem.system.cpp

```diff
@@ -160,6 +160,14 @@ namespace eosiosystem {
       _gstate.min_account_stake = min_account_stake;
    }
 
+   uint64_t system_contract::get_min_account_stake() {
+      eosio::remprice_idx remprice_table(oracle_account, oracle_account.value);
+      auto it = remprice_table.find(rem_usd_pair.value);
+      bool is_valid_price = (it != remprice_table.end()) && ((current_time_point() - it->last_update.to_time_point()) <= eosio::minutes(70));
```

Comment:
70 minutes should be parametrized, let's avoid magic numbers

Reply:
@roman-tik, done.